.copr/Makefile

```diff
@@ -2,6 +2,8 @@ include Makefile
 
 SOURCEDIR := $(shell rpmbuild -E %_sourcedir)
 SRCRPMDIR := $(shell rpmbuild -E %_srcrpmdir)
+$(shell mkdir -p $(SOURCEDIR))
```

Comment:
This feels like a misuse of Makefile features. Unconditionally executing shell commands when the Makefile is parsed. Shouldn't `$(SOURCEDIR)` and `$(SRCRPMDIR)` become targets, and the `mkdir` command become the recipe? Something like this?

```make
$(SOURCEDIR) $(SRCRPMDIR):
    mkdir -p $@
```